nebula-logger/plugins/async-failure-additions/plugin/customMetadata/LoggerParameter.FlowExecutionErrorEventHandled.md-meta.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<CustomMetadata
+    xmlns="http://soap.sforce.com/2006/04/metadata"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+>
+    <label>Is FlowExecutionErrorEvent Handled?</label>
+    <protected>true</protected>
+    <values>
+        <field>Description__c</field>
+        <value xsi:type="xsd:string">1 if enabled, 0 if disabled. True values create Nebula Logger entries for unhandled flow errors.</value>
+    </values>
+    <values>
+        <field>Value__c</field>
+        <value xsi:type="xsd:string">0</value>
```

Comment:
Shouldn't this be a value of 'true' or 'false' (instead of '1' or '0')? It looks like the code is treating it as a boolean.

Reply:
@jongpie that's what I initially thought as well, and I swear I had an error using those string values, but now everything seems to be working as intended so I will swap to true/false.